### ISSUE TITLE
Background Jobs

### ISSUE THREAD

**Issue by ian** [rel-js/rel#9]
Comparisons

- https://blitzjs.com/docs/background-processing-with-quirrel#reminding-a-user-of-an-upcoming-event
- https://repeater.dev/
- https://demo.quirrel.dev/

**Comment by thelinuxlich** [comment]
We may replicate repeater SaaS using https://github.com/OptimalBits/bull

Then we could have a `jobs` folder where the user adds functions to run on a specified interval, repeatable or not

**Comment by ian** [comment]
I used bull previously (specifically bull MQ), works pretty well and was my favorite of the redis-based job queues.

I think it wouldn't be hard to integrate with our current architecture since I have a version of this running on fastify already.

**Comment by thelinuxlich** [comment]
No, it wouldn't be hard, and I loved the GraphQL API to inspect jobs, we would reimplement this too
